app/templates/components/public/session-filter.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="ui secondary menu">
+  <a href="#" class="item {{if (eq selectedTrack 'all') 'active'}}" id="all" onclick={{action 'filter' 'All'}}>
```

Comment:
Change to 

```hbs
<a href="#" class="item {{unless selectedTrackId 'active'}}" {{action 'filter'}}>
```

Here if `selectedTrackId` is null it is considered to be all tracks. 
And for action you don't have to specify `onclick=`
And note that we're not passing any argument to the action. hence trackId in the action will be null.